Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2_test.py

```diff
@@ -11,11 +18,67 @@
+     'entityB': 'md5', 'entityBFamily': 'Indicator', 'entityBType': 'File', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence', 'entityA': 'file', 'entityAFamily': 'Indicator', 'entityAType': 'File',
+     'entityB': 'md5', 'entityBFamily': 'Indicator', 'entityBType': 'File', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'}
+]
+
+EMAIL_RELATIONSHIP = [
+    {'brand': 'Cofense Intelligence', 'entityA': 'email@email.com', 'entityAFamily': 'Indicator',
+     'entityAType': 'Email', 'entityB': 'email@email.com', 'entityBFamily': 'Indicator',
+     'entityBType': 'Email', 'fields': {}, 'name': 'related-to', 'reverseName': 'related-to',
+     'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence', 'entityA': 'email@email.com', 'entityAFamily': 'Indicator',
+     'entityAType': 'Email', 'entityB': 'md5', 'entityBFamily': 'Indicator', 'entityBType': 'File', 'fields': {},
+     'name': 'related-to', 'reverseName': 'related-to', 'type': 'IndicatorToIndicator'}]
+
+URL_RELATIONSHIP = [
+    {'brand': 'Cofense Intelligence', 'entityA': 'url', 'entityAFamily': 'Indicator', 'entityAType': 'URL',
+     'entityB': 'url', 'entityBFamily': 'Indicator', 'entityBType': 'Domain Name', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence', 'entityA': 'url', 'entityAFamily': 'Indicator', 'entityAType': 'URL',
+     'entityB': 'url', 'entityBFamily': 'Indicator', 'entityBType': 'Domain Name', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence', 'entityA': 'url',
+     'entityAFamily': 'Indicator', 'entityAType': 'URL', 'entityB': 'url', 'entityBFamily': 'Indicator',
+     'entityBType': 'IPv4 Address', 'fields': {}, 'name': 'related-to', 'reverseName': 'related-to',
+     'type': 'IndicatorToIndicator'}]
+
+IP_RELATIONSHIP = [
+    {'brand': 'Cofense Intelligence', 'entityA': '127.0.0.1', 'entityAFamily': 'Indicator', 'entityAType': 'IP',
+     'entityB': 'url', 'entityBFamily': 'Indicator', 'entityBType': 'URL', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence', 'entityA': '127.0.0.1', 'entityAFamily': 'Indicator', 'entityAType': 'IP',
+     'entityB': 'md5', 'entityBFamily': 'Indicator', 'entityBType': 'File', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'},
+    {'brand': 'Cofense Intelligence',
+     'entityA': '127.0.0.1', 'entityAFamily': 'Indicator', 'entityAType': 'IP', 'entityB': 'md5',
+     'entityBFamily': 'Indicator', 'entityBType': 'File', 'fields': {}, 'name': 'related-to',
+     'reverseName': 'related-to', 'type': 'IndicatorToIndicator'}]
```

Comment:
after changing the create_relationship function (see comment in function regarding entityBType), change these accordingly